app/javascript/dashboard/components-next/input/Input.vue

```diff
@@ -85,15 +120,25 @@ const handleInput = event => {
     <slot name="prefix" />
     <input
       :id="id"
+      ref="inputRef"
       :value="modelValue"
-      :class="[customInputClass, inputBorderClass]"
+      :class="[
+        customInputClass,
+        inputBorderClass,
+        {
+          error: messageType === 'error',
+          focus: isFocused,
```

Comment:
Why do we need these classes here?

Reply:
There is some case where we don't need to show the border, but we need to show focus or error border state. That's the reason.